diff --git a/src/lib/missing/endian.h b/src/lib/missing/endian.h
similarity index 83%
rename from src/lib/portable_endian.h
rename to src/lib/endian.h
index 99845d0..0a3f4c2 100644
--- a/src/lib/missing/endian.h
+++ b/src/lib/missing/endian.h
@@ -1,9 +1,3 @@
-// "License": Public Domain
-// I, Mathias Panzenböck, place this file hereby into the public domain. Use it at your own risk for whatever you like.
-// In case there are jurisdictions that don't support putting things in the public domain you can also consider it to
-// be "dual licensed" under the BSD, MIT and Apache licenses, if you want to. This code is trivial anyway. Consider it
-// an example on how to get the endian conversion functions on different platforms.
-
 #ifndef PORTABLE_ENDIAN_H__
 #define PORTABLE_ENDIAN_H__
 
 
